File: tests/fixtures/multilang/sample.py
import os
from pathlib import Path

MAX_RETRIES = 3

def greet(name: str) -> str:
    return f"Hello, {name}"

def add(a: int, b: int) -> int:
    return a + b

class Config:
    def __init__(self, host: str, port: int):
        self.host = host
        self.port = port

    def url(self) -> str:
        return f"http://{self.host}:{self.port}"

File: tests/fixtures/multilang/sample.ts
import { Request, Response } from 'express';

export const MAX_RETRIES = 3;

export function greet(name: string): string {
    return `Hello, ${name}`;
}

export class Config {
    public host: string;
    public port: number;

    constructor(host: string, port: number) {
        this.host = host;
        this.port = port;
    }

    url(): string {
        return `http://${this.host}:${this.port}`;
    }
}

export interface Serializable {
    serialize(): string;
}

export type StringOrNum = string | number;
